src/workerd/jsg/rtti.h

```diff
@@ -249,7 +250,8 @@ FOR_EACH_NUMBER_TYPE(DECLARE_NUMBER_TYPE)
   F(ByteString)                                                                                    \
   F(USVString)                                                                                     \
   F(DOMString)                                                                                     \
-  F(jsg::JsString)
+  F(jsg::JsString)                                                                                 \
+  F(::rust::String)
```

Comment:
Should likely add to the rtti-test coverage to ensure the `::rustString` is correctly handled.